scripts/ci-entrypoint.sh

```diff
@@ -305,9 +303,10 @@ on_exit() {
     unset KUBECONFIG
     go run -tags e2e "${REPO_ROOT}"/test/logger.go --name "${CLUSTER_NAME}" --namespace default
     "${REPO_ROOT}/hack/log/redact.sh" || true
-    # cleanup
-    if [[ -z "${SKIP_CLEANUP:-}" ]]; then
-        cleanup
+    # cleanup all resources we use
+    if [[ ! "${SKIP_CLEANUP:-}" == "true" ]]; then
+        "${KUBECTL}" --kubeconfig "${REPO_ROOT}/${KIND_CLUSTER_NAME}.kubeconfig" delete cluster "${CLUSTER_NAME}" --wait=true || echo "Unable to delete cluster ${CLUSTER_NAME} nodes"
```

Comment:
```suggestion
        "${KUBECTL}" --kubeconfig "${REPO_ROOT}/${KIND_CLUSTER_NAME}.kubeconfig" delete cluster "${CLUSTER_NAME}" --wait=true || echo "Unable to delete cluster ${CLUSTER_NAME}"
```

we're deleting the entire cluster, not sure the word nodes is needed here

Reply:
ACK